src/components/image/index.jsx

```diff
@@ -57,11 +46,13 @@ const Image = ({
 			.map((responsiveImageWidth) =>
 				// divide the derived aspect ratio by each of the responsiveImages widths to get the height
 				// aspect ratio of zero will not show the height
-				srcWithOptionsWithoutHeightWidth.concat(
-					`&width=${responsiveImageWidth}`,
-					`${aspectRatio !== 0 && height ? `&height=${responsiveImageWidth / aspectRatio}` : ""}`,
-					` ${responsiveImageWidth}w`
-				)
+
+				formatSrc(
+					resizerURL.concat(src),
+					resizedOptions,
+					responsiveImageWidth,
+					aspectRatio !== 0 && height ? Math.floor(responsiveImageWidth / aspectRatio) : undefined
```

Comment:
The `Math.floor` should be put within the `formatSrc` function that way we will not have situations where it's different between components or consumers of the `formatSrc` function

Reply:
Oh right, good point.  Sorry @ecarlisle